packages/react-ui-components/src/SelectBox_Option_MultiLineWithThumbnail/style.css

```diff
@@ -1,8 +1,25 @@
 .multiLineWithThumbnail__item {
+    display: grid;
+    grid-template-columns: auto 1fr;
+    gap: -var(--spacing-Half);
```

Comment:
A negative gap is marked as invalid value and also doesn't seem to have any effect.

The styling would also be simpler when the gap is actually used and the padding of the element disabled and therefore the image correctly positioned (also the image margin needs to be disabled.

The result is looking the same but without all the CSS pushing around.